### ISSUE TITLE
question in training data

### ISSUE THREAD

**Issue by chenlihuang** [vlievin/Kaggle-Leaf-classification-using-RNNs#1]
In your train.csv the label in the data has no label for class, can you help me  ?or  my  understanding has  some bias?

**Comment by vlievin** [comment]
Hi, the label is the species name of the leaf. So, the labels correspond to the "species" column in the training set. 
This is a project for a Kaggle competition (Leafs classification).
